packages/flutter_tools/test/general.shard/runner/runner_test.dart

```diff
@@ -74,6 +78,58 @@ void main() {
       ProcessManager: () => FakeProcessManager.any(),
       Usage: () => CrashingUsage(),
     });
+
+    testUsingContext('GitHub issue template', () async {
+      const String similarURL = 'https://example.com/1';
+      when(mockGitHubTemplateCreator.toolCrashSimilarIssuesGitHubURL(any))
+        .thenAnswer((_) async => similarURL);
+      const String templateURL = 'https://example.com/2';
+      when(mockGitHubTemplateCreator.toolCrashIssueTemplateGitHubURL(any, any, any, any, any))
+        .thenAnswer((_) async => templateURL);
+      final Completer<void> completer = Completer<void>();
+      // runner.run() asynchronously calls the exit function set above, so we
+      // catch it in a zone.
+      unawaited(runZoned<Future<void>>(
+          () {
```

Comment:
indentation: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#indent-multi-line-argument-and-parameter-lists-by-2-characters